src/System.Private.Xml/src/System/Xml/Core/XmlTextWriter.cs

```diff
@@ -1310,8 +1347,7 @@ private void InternalWriteEndElement(bool longFormat)
                     {
                         Indent(true);
                     }
-                    _textWriter.Write('<');
-                    _textWriter.Write('/');
+                    _textWriter.Write(s_selfClosingTagOpen);
```

Comment:
StreamWriter.Write(char[]) is faster than StreamWriter.Write(string).

---
In reply to: [122563818](https://github.com/dotnet/corefx/pull/21155#discussion_r122563818) [](ancestors = 122563818)